che-plugin-viewer/src/imgs/logos/stackoverflow.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 25 32" style="enable-background:new 0 0 25 32;" xml:space="preserve">
```

Comment:
Do we have rights to use all these images?

Reply:
I will clear all the unnecessary images and logos in my next PR.